VersionControlUI/src/DiffManager.cpp

```diff
@@ -94,9 +94,11 @@ DiffSetup DiffManager::initializeDiffPrerequisites(QString oldVersion, QString n
 
 	// load newer version
 	diffSetup.newVersionManager_ = createTreeManagerFromVersion(diffSetup.repository_, diffSetup.newVersion_);
+	diffSetup.newVersionManager_->setRevisionName("new");
```

Comment:
This is independent from the saving of reviews. It is used during the load of a saved review to assign the comments to nodes of the old or new tree manager depending on this attribute.